docs/plugins/integrations/snmp.asciidoc

```diff
@@ -14,8 +14,123 @@ START - GENERATED VARIABLES, DO NOT EDIT!
 END - GENERATED VARIABLES, DO NOT EDIT!
 ///////////////////////////////////////////
 
-[id="plugins-{type}s-{plugin}"]
+// include::{include_path}/plugin_header.asciidoc[]
 
-=== SNMP Integration Plugin
+experimental[]
 
-COMING SOON!
+.Technical Preview
+****
+The new `integration-snmp` plugin, and its component plugins--`input-snmp` and `input-snmptrap`--are available in _Technical Preview_ at {ls} 8.14.0.
```

Comment:
Note to self @karenzone: Get exact versioning right, and determine how we want to handle this for minor and patch releases (lockfile).